packages/trader/src/Modules/Trading/Containers/trade-params.tsx

```diff
@@ -17,14 +16,21 @@ import { observer } from '@deriv/stores';
 import { useTraderStore } from 'Stores/useTraderStores';
 import Fieldset from 'App/Components/Form/fieldset.jsx';
 
-const TradeParams = observer(({ is_minimized }) => {
+type TTradeParams = {
+    is_minimized: boolean;
+};
+
+const TradeParams = observer(({ is_minimized }: TTradeParams) => {
     const { form_components } = useTraderStore();
-    const isVisible = component_key => {
+    const isVisible = (component_key: string) => {
         return form_components.includes(component_key);
     };
     return (
         <React.Fragment>
-            {isVisible('duration') && <Duration key={'duration'} is_minimized={is_minimized} />}
+            {isVisible('duration') && (
+                // @ts-expect-error: we expect error here because <Duration /> is not in TS yet
```

Comment:
Mark all ts ignore with TODO. It is easier to track for refactoring in the future